docs/src/site/markdown/index.md.vm

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
I wonder if we should use two commits to somehow force the git history to see this file as a rename of `index.apt.vm`.

Reply:
This PR has two commits for exactly that reason